doc/userguide/rules/http-keywords.rst

```diff
@@ -728,6 +728,8 @@ file.data
 
 With ``file.data``, the HTTP response body is inspected, just like
 with ``http.response_body``. The ``file.data`` keyword is a sticky buffer.
+``file.data`` also works for HTTP request body (and can be used in other
+protocols than HTTP1).
```

Comment:
I think "and can be used in protocols other than HTTP1" sounds better

Reply:
ok